python/lsst/meas/algorithms/skyObjects.py

```diff
@@ -138,11 +138,27 @@ def generateSkyObjects(mask, seed, config):
 
 
 class SkyObjectsTask(Task):
-    """Generate a list of Footprints of sky objects.
+    """Generate a list of Footprints of sky sources (regions on the sky that
+    do not otherwise have detections).
```

Comment:
"Generate a list of Footprints of sky objects/sources (regions on the sky that do not otherwise have detections)."